azure-spring-boot-samples/azure-active-directory-spring-boot-sample/src/main/resources/static/App/Scripts/adal-angular.js

```diff
@@ -212,6 +212,9 @@ if (typeof module !== 'undefined' && module.exports) {
                     getCachedToken: function (resource) {
                         return _adal.getCachedToken(resource);
                     },
+                    getInstance:function (resource) {
```

Comment:
do we want to merge our changes to adal js to https://github.com/AzureAD/azure-activedirectory-library-for-js?

Reply:
I was not aware of  above repository, will check.

Reply:
no worries. this is the ajs lib we use on the client side.